### PR TITLE
Adform adapter crid & price type

### PR DESCRIPTION
**Pull request by volhalink**

### CODE REVIEW

File: adapters/adform/adform.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ type adformRequest struct {
 
 type adformAdUnit struct {
 	MasterTagId json.Number `json:"mid"`
+    PriceType string `json:"priceType,omitempty"`
```
Comment (vstatkevich) [inline adapters/adform/adform.go]: need to fix formatting (try run with GoLand: `Tools`->`Go tools`->`Go fmt project`)

File: adapters/adform/adform.go [hunk 2]
```diff
@@ -201,6 +205,15 @@ func (r *adformRequest) buildAdformUrl(a *AdformAdapter) string {
 	return fmt.Sprintf("%s/?CC=1&rp=4&fd=1&stid=%s&ip=%s%s&%s", uri, r.tid, r.ip, adid, strings.Join(adUnitsParams, "&"))
 }
 
+func getValidPriceTypeParameter(adUnit *adformAdUnit) string {
+	priceTypeParameter := ""
+	_, valid := validPriceTypes[adUnit.PriceType]
```
Comment (vstatkevich) [inline adapters/adform/adform.go]: make type comparison case-insensitive (e.g. net, NeT, Net)

File: adapters/adform/adform.go [hunk 3]
```diff
@@ -293,6 +306,12 @@ func openRtbToAdformRequest(request *openrtb.BidRequest) (*adformRequest, []erro
 			secure = true
 		}
 
+		_, valid := validPriceTypes[adformAdUnit.PriceType]
```
Comment (vstatkevich) [inline adapters/adform/adform.go]: func `pbsRequestToAdformRequest` should use the same validation

File: adapters/adform/adform.go [hunk 4]
```diff
@@ -354,6 +373,7 @@ func toOpenRtbBids(adformBids []*adformBid, r *openrtb.BidRequest) []*adapters.T
 			W:      bid.Width,
 			H:      bid.Height,
 			DealID: bid.DealId,
+			CrID:   bid.CreativeId,
```
Comment (vstatkevich) [inline adapters/adform/adform.go:424]: func `toPBSBidSlice` should have the same assignment `Creative_id: bid.CreativeId,`

Comment (volhalink) [reply adapters/adform/adform.go:424]: added, thank you

**Review by vstatkevich — APPROVED** [review]

**Pull request closed without merging**